src/clusterfuzz/_internal/platforms/android/adb.py

```diff
@@ -420,13 +420,13 @@ def start_cuttlefish_device(use_kernel=False):
   device_memory_mb = environment.get_value('DEVICE_MEMORY_MB',
                                            DEFAULT_DEVICE_MEMORY_MB)
   launch_cvd_command_line = (
-      f'{launch_cvd_path} -daemon -memory_mb {device_memory_mb} '
-      '-report_anonymous_usage_stats Y')
+      f'sudo {launch_cvd_path} --daemon --memory_mb={device_memory_mb} '
```

Comment:
Is backwards compatibiltiy a concern here? i.e. would  we be running this on devices/configurations  where we  need the old cmdline style?

Reply:
I can confirm that all of our cuttlefish instances (150 pre / 50 normal) launch via this command line, neither physical device nor hosts will reach this section of code. There should be no concerns of retro-compat.

After pair programming with Hamzeh@, we discovered that these flags were originally implemented incorrectly and have never been consumed since launch. The command in '{cmd}' succeeds, but the options are not consumed.

Changes in upstream cuttlefish code cause this implementation to fail, leading out current virtual device fleet in a complete stalemate.